backend/apps/users/models.py

```diff
@@ -53,6 +45,18 @@ def generate_email_verification_code(self):
             {'verification_code': verification.code}
         )
 
+class Doctor(User):
+    specialty = models.CharField(max_length=50)
+    scr_access = models.BooleanField()
+    online_consultation_capability = models.BooleanField()
+    
+class Patient(User):
+    age = models.PositiveIntegerField()
+    address = models.CharField(max_length=200)
+    
+class PharmacyUser(User):
```

Comment:
This model is not related with USER  and add some important fields are not added like postal code. check and add all the fields